build-test.sh

```diff
@@ -106,67 +111,68 @@ wait_for_renter_upload_ready() {
       sleep 1
     done"
   
-  # execute
-  doing "Waiting for renter to become upload ready at $url"
-  timeout $timeout sh -c "$cmd"
-  finished
+  # Execute
+  log_cmd "Waiting for renter to become upload ready at $url" timeout $timeout sh -c "$cmd"
 }
 
 # Remove test containers before we start
 cleanup_test_containers
 
 # Iterate over all Dockerfiles
-for DIR in ./
+for DIR in .
 do
   # Build the image
-  docker build \
+  log_cmd "Building a test image according to $DIR/Dockerfile" docker build \
     --no-cache \
     --tag sia-ant-farm-image-test \
     -f $DIR/Dockerfile \
     .
 
-  # Test with a single published port
+  # Test with a single published standard renter port
   # Run container in detached state
   DUMMY_DATA_DIR=$(mktemp -d)
-  doing "Starting test container"
-  docker run \
+  TEST1_RENTER_PORT=9980
```

Comment:
Do we care that this coincides with the usual port? I don't think so but it used to be 9988, so I'm asking just in case.

Reply:
Previously there was `9988` port to check that we can forward from docker inner `9980` port to any other port.
I have updated the test script to check we can use default port and to check non-standard port.

I think it's ok to use usual port.
If somebody is affected at his localhost he can change it locally, the Travis CI is more important and there it is ok.